src/components/project.ts

```diff
@@ -209,4 +223,30 @@ export class Project extends pulumi.ComponentResource {
     );
     this.services[options.serviceName] = service;
   }
+
+  private createMongoService(options: MongoService) {
+    if (!this.cluster) return;
```

Comment:
What do you think about trowing an error if the cluster does not exist instead of early return?